products/urls.py

```diff
@@ -1,9 +1,10 @@
 from django.urls import path
 
-from products.views import CategoryView,ProductView
+from products.views import CategoryView,ProductView,ProductFilterView
 
 urlpatterns = [
     path('/category', CategoryView.as_view()),
     path('/product', ProductView.as_view()),
```

Comment:
이제와서 발견한건데 이미 /products 로 시작하는 경로아닌가요? 그럼 그냥 중복안되게 '' 로 주는게 나을것 같습니다.
```suggestion
    path('', ProductView.as_view()),
```

Reply:
list로 수정했습니다~